recipes/opam/meta.yaml

```diff
@@ -0,0 +1,39 @@
+#{% set version = "2.0.0-beta3" %}
+{% set unixsha256 = "6b560303316bc70099116ed3c5bfcc59928b9f8e10a99d994ea98ca6dad359d5" %}
+{% set osxsha256 = "6b560303316bc70099116ed3c5bfcc59928b9f8e10a99d994ea98ca6dad359d5" %}
+
+
+package:
+  name: opam
+  version: 2.0.0.beta3
+
+source:
+  fn: opam-2.0.0-beta3-x86_64-Linux
+  url: https://github.com/ocaml/opam/releases/download/2.0.0-beta3/opam-2.0.0-beta3-x86_64-Linux
+  sha256: {{ unixsha256 }}  # [unix]
+  fn: opam-2.0.0-beta3-x86_64-Darwin
+  url: https://github.com/ocaml/opam/releases/download/2.0.0-beta3/opam-2.0.0-beta3-x86_64-Darwin
+  sha256: {{ osxsha256 }}  # [osx]
+
+build:
+  number: 0
+  skip: True  # [win] 
+
+#requirements:
+#  run:
+#  - gmp >=5.0.1,<7
+
+test:
+  commands:
+    - opam --version
+
+about:
+  home: https://opam.ocaml.org/
+  license: LGPL-2.1
```

Comment:
There should also be a `license_file` entry that points to license file in the source repository. In most cases that is just whatever name the license file has in the source repository (e.g. `LICENSE`, but check the opam repo to be sure).